less/events.less

```diff
@@ -66,28 +86,34 @@
     height: 47px;
     color: white;
     border: none;
+    padding-right: 2vh;
+    padding-left: 2vh;
 }
 
 .checkboxList {
     background-color: white;
     list-style-type: none;
-    padding-left: 10px;
+    padding: 2vh 0 2vh 1vh;
```

Comment:
Is there a reason why you are using vh units here? I'm not against the idea but we usually use px for everything.